unified-installer/astra-unified-installer.sh

```diff
@@ -2478,11 +2528,10 @@ step_generate_trident_operator_patch() {
         if echo "$_EXISTING_TRIDENT_OPERATOR_PULL_SECRETS" | grep -q "^${IMAGE_PULL_SECRET}$" &> /dev/null; then
             logdebug "image pull secret '$IMAGE_PULL_SECRET' already present in trident-operator"
         else
-            local -r secret_obj='{"name": "'"$IMAGE_PULL_SECRET"'"}'
             if [ -z "$_EXISTING_TRIDENT_OPERATOR_PULL_SECRETS" ]; then
-                patch_list+=',{"op":"replace","path":"/spec/template/spec/imagePullSecrets","value":['"$secret_obj"']}'
```

Comment:
This is just for consistency. We don't have this $secret_obj variable in step_generate_torc_patch(), so I'd rather we do the same thing in both places. This came up with I was solving a merge conflict, otherwise I would not have even thought to touch this.